### PR TITLE
2.x: Remove fromEmitter() in wiki

### PR DESCRIPTION
**Pull request by CoXier**
Remove the mentions of `fromEmitter` from the `Creating-Observables.md` wiki page.

### CODE REVIEW

File: docs/Creating-Observables.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ This page shows methods that create Observables.
 * [**`just( )`**](http://reactivex.io/documentation/operators/just.html) — convert an object or several objects into an Observable that emits that object or those objects
 * [**`from( )`**](http://reactivex.io/documentation/operators/from.html) — convert an Iterable, a Future, or an Array into an Observable
 * [**`create( )`**](http://reactivex.io/documentation/operators/create.html) — **advanced use only!** create an Observable from scratch by means of a function, consider `fromEmitter` instead
```
Comment (akarnokd) [inline docs/Creating-Observables.md]: Please remove the "consider `fromEmitter` instead" from this line as well.

**Comment by akarnokd** [comment]
You should not keep the welcome text and please provide some reasonable description about what you contribute next time. This text is as much for the reviewers of your PR as is for the search engines.

File: docs/Creating-Observables.md (hunk 1 above)
Comment (CoXier) [reply docs/Creating-Observables.md]: It's my mistake. I'm sorry for that.

**Comment by CoXier** [comment]
Thanks for your suggestion. @akarnokd

**Review by akarnokd — APPROVED** [review]

**Pull request merged**